### DISCUSSION TITLE
四月更新 - 1

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#949] [通知]
## 🚀 功能

- 优化播放控制器操作行为 @Richasy (#940)

## 🐛 Bug 修复

- 修复全屏按钮在退出全屏模式后依然被选中的问题 @Richasy (#948)
- 修复用户视频滚动中条目消失的问题 @Richasy (#946)
- 修复推荐视频卡片中的元素遮挡问题 @Richasy (#945)
- 修复后台会连播关联视频的问题 @Richasy (#944)
- 修复下载文件夹选择器在Win10上不工作的问题 @Richasy (#943)
- 修复稍后再看条目菜单显示异常的问题 @Richasy (#942)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2204.1.0'>四月更新 - 1</a>.</em>

**Comment by xyl23333** [comment]
啊哈哈哈哈，更新来咯～

**Reply by blibilijojo** [reply]
啥时候更新啊，明天？还有，我最期待的控制器优化终于来了啊啊啊啊啊，我受了这个气半年了

**Comment by Alpha-lab1** [comment]
很严重的一个bug，鼠标停留在哔哩页面上时会莫名消失

**Reply by xyl23333** [reply]
我也刚刚遇到了，好像up主历史视频页面也没有鼠标指针。

**Reply by blibilijojo** [reply]
我提交了，可以去看看原因，晚上突然悟到了原因

**Reply by blibilijojo** [reply]
> 我也刚刚遇到了，好像up主历史视频页面也没有鼠标指针。

刚才一看，修复了
